src/language-js/parse/postprocess/typescript.js

```diff
@@ -73,9 +73,10 @@ function throwErrorForInvalidNodes(ast, options) {
     if (esTreeNode !== node) {
       return;
     }
+
     throwErrorForInvalidDecorator(tsNode, esTreeNode, tsNodeToESTreeNodeMap);
     throwErrorForInvalidAbstractProperty(tsNode, esTreeNode);
   });
 }
 
-module.exports = { throwErrorForInvalidNodes };
+module.exports = { throwErrorForInvalidNodes, throwSyntaxError };
```

Comment:
Can we move `throwSyntaxError` to other file like a `parse/postprocess/throw-syntax-error.js`?